search-commons/src/test/java/no/unit/nva/search/ticket/TicketClientByOwnerTest.java

```diff
@@ -72,16 +74,22 @@ static void beforeAll() throws IOException {
     var cachedJwtProvider = setupMockedCachedJwtProvider();
     ticketClient = new TicketClient(HttpClient.newHttpClient(), cachedJwtProvider);
 
-    var restClientBuilder = RestClient.builder(HttpHost.create(container.getHttpHostAddress()));
-    var restHighLevelClientWrapper = new RestHighLevelClientWrapper(restClientBuilder);
-    var indexingClient = new IndexingClient(restHighLevelClientWrapper, cachedJwtProvider);
+    RestClientBuilder restClientBuilder = null;
+    try {
+      restClientBuilder = RestClient.builder(HttpHost.create(container.getHttpHostAddress()));
```

Comment:
Could we merge the declaration and creation here? I don't see a reason to split them.